packages/pwa/app/pages/home/index.jsx

```diff
@@ -25,14 +25,18 @@ import Hero from '../../components/hero'
 import Seo from '../../components/seo'
 import Section from '../../components/section'
 import {heroFeatures, features} from './data'
+import ProductScroller from '../../components/product-scroller'
+
+const SHOP_PRODUCTS_CATEGORY_ID = 'newarrivals'
```

Comment:
We have other values similar to this one for the PLP in the constants file located at `app/constants.js`. Although I don't see this constant ever being a configurable option in our new app config proposal, it might make more sense to be consistent (😅) with the placement